python/grass/temporal/core.py

```diff
@@ -374,13 +374,29 @@ def get_tgis_version():
 
 
 def get_tgis_db_version():
-    """Get the version number of the temporal framework
-    :returns: The version number of the temporal framework as string
+    """Get the supported version number of the temporal database
```

Comment:
See the other "supported" comment, e.g., grass/temporal/metadata.py seems to support 2 and 3. Also, if you want to add an adjective, you should probably drop the word number.

Reply:
Done by 4d0294a10